.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,6 +9,13 @@ assignees: ''
 
 Provide as much information as possible. At least, this should include a description of your issue and steps to reproduce the problem. If possible also provide a summary of what steps or workarounds you have already tried.
 
+### System information
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
```

Comment:
CUDA version if applicable? I'm not sure how easy it is to query but it comes up quite a bit

Reply:
Good idea, added it.